src/components/Search/SearchBar.jsx

```diff
@@ -95,11 +95,11 @@ export class SearchBar extends React.Component {
               <div className="advanced-container flex-fixed">
               {
                this.state.advancedShown ?
-                 <button className="a-btn a-btn__link o-expandable_cue-close"
+                  <button className="a-btn a-btn__link"
                       onClick={ this._onAdvancedClicked }>
                       Hide advanced search tips
                   </button> :
-                  <button className="a-btn a-btn__link o-expandable_cue-open"
+                  <button className="a-btn a-btn__link"
```

Comment:
looks like the only reason why this button was set in the ternary was because of the o-expandable_cue-open classes.  maybe combine this into a single button and only have it modify the Show/Hide inner text?

Reply:
Good call! Updated!